packages/error/tsconfig.json

```diff
@@ -3,6 +3,7 @@
   "compilerOptions": {
     "outDir": "./dist",
     "rootDir": "./src",
-    "types": ["mocha", "node"]
+    "types": ["mocha", "node"],
+    "lib": ["ES2022"],
```

Comment:
Is this still necessary? I would expect this value to cascade down from `tsconfig.base.json`, which `tsconfig.package.json` extends.

Reply:
I guess it's no longer needed since we recently updated to ES2022 in `tsconfig.base.json` in #8678. Removed in ac43af686146050016ea7970edf6fb7a63f15174